src/test/resources/org/prebid/server/it/openrtb2/algorix/test-auction-algorix-request.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": "tid",
+  "imp": [
+    {
+      "id": "impId001",
+      "banner": {
+        "format": [
+          {
+            "w": 300,
+            "h": 250
+          }
+        ]
+      },
+      "ext": {
+        "algorix": {
+          "sid": "testSid",
+          "token": "testToken"
+        }
+      }
+    }
+  ],
+  "at": 1,
+  "tmax": 5000,
+  "cur": [
+    "USD"
+  ],
```

Comment:
We prefer to keep integration tests as simple as possible so, this can be removed here, and then bid-request and auction-response files tmax should be updated to 2000, as it is default value